packages/admin/src/brevoConfiguration/BrevoConfigForm.tsx

```diff
@@ -174,6 +180,37 @@ export function BrevoConfigForm({ scope }: FormProps): React.ReactElement {
                                 label={<FormattedMessage id="cometBrevoModule.brevoConfig.sender" defaultMessage="Sender" />}
                                 fullWidth
                             />
+                            <NumberField
+                                name="folderId"
+                                defaultValue={1}
+                                label={
+                                    <>
+                                        <FormattedMessage id="cometBrevoModule.brevoConfig.folderId" defaultMessage="Folder ID" />
+                                        <Tooltip
+                                            title={
+                                                <FormattedMessage
+                                                    id="cometBrevoModule.brevoConfig.folderId.info"
+                                                    defaultMessage="By default, the folder ID should be set to 1 unless you have specifically configured another folder in Brevo."
+                                                />
+                                            }
+                                            sx={{ marginLeft: "5px" }}
+                                        >
+                                            <Info />
+                                        </Tooltip>
+                                    </>
+                                }
+                                fullWidth
+                                validate={(value) => {
+                                    if (typeof value !== "number") {
+                                        return (
+                                            <FormattedMessage
+                                                id="cometBrevoModule.brevoConfig.folderId.validation"
+                                                defaultMessage="Please enter a number"
+                                            />
+                                        );
+                                    }
+                                }}
```

Comment:
I would move the validation in it's own function (outside the JSX) for better readability

Reply:
Done here: 9cfd6671ad412ef23e77650c2cb8376ef1d676f7